### PR TITLE
13.0 mig sentry

### PR DESCRIPTION
**Pull request by IvanShirokikh**
Simple migration sentry

### CODE REVIEW

File: sentry/README.rst [hunk 1]
```diff
@@ -0,0 +1,177 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======
+Sentry
+======
+
+This module allows painless `Sentry <https://sentry.io/>`__ integration with
```
Comment (luisg123v) [inline sentry/README.rst:28]: README should be split according to [OCA's template module](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by newtratip** [comment]
@IvanShirokikh Could you fix for conflict file ?

**Comment by tsabi** [comment]
Hi @IvanShirokikh!
The raven module is deprecated in favor of sentry-sdk package: https://docs.sentry.io/platforms/python/
Is it possible to do the transition now? Can i help?
BR,
Csaba

File: sentry/README.rst (hunk 1 above)
Comment (IvanShirokikh) [reply sentry/README.rst:28]: Ok i tried to fix it

**Comment by IvanShirokikh** [comment]
> @IvanShirokikh Could you fix for conflict file ?

Resolved

**Comment by IvanShirokikh** [comment]
Hi @tsabi.
Yes i know about it.
Sorry, I am absolutely not educated in terms of open source interaction, probably you could offer your PR with a new library.

**Comment by IvanShirokikh** [comment]
@luisg123v Are there any other problems to accept this PR?
Migration to sentry_sdk will take some time, some functions that were used from raven, such as: fetch_git_sha, get_environ and get_headers, were removed from there.
Also there are changed approaches to tracking logging and filtering sensitive data.
There is currently no version of sentry for Odoo 13 at all, and in a few months there will already be version 14

**Review by luisg123v — APPROVED** [review]
@moylop260, sentry for v13

**Comment by moylop260** [comment]
I will merge it manually since that there is an issue in the readme.rst generation.
 - https://github.com/OCA/maintainer-tools/issues/459

File: sentry/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Other `client arguments
+<https://docs.sentry.io/clients/python/advanced/#client-arguments>`_ can be
+configured by prepending the argument name with *sentry_* in your Odoo config
+file. Currently supported additional client arguments are: ``install_sys_hook,
+include_paths, exclude_paths, machine, auto_log_stacks, capture_locals,
+string_max_length, list_max_length, site, include_versions, environment``.
+
+Example Odoo configuration
+--------------------------
```
Comment (sbidoul) [inline sentry/readme/CONFIGURE.rst:66]: You should use `~` as underline here. cc/ @moylop260 (from https://github.com/OCA/maintainer-tools/issues/459)

Comment (moylop260) [reply sentry/readme/CONFIGURE.rst:66]: Hi @sbidoul 
Thanks for check it
Thank you
I will fix it

**Comment by moylop260** [comment]
Fixed from:
 - https://github.com/OCA/server-tools/pull/1845

**Pull request merged**